<map id="pytest_userver.plugins.log_capture.ClientConnectTimeoutError" name="pytest_userver.plugins.log_capture.ClientConnectTimeoutError">
<area shape="rect" id="Node000001" title=" " alt="" coords="5,167,264,207"/>
<area shape="rect" id="Node000002" href="$df/d4b/classpytest__userver_1_1plugins_1_1log__capture_1_1BaseError.html" title=" " alt="" coords="38,79,231,119"/>
<area shape="poly" id="edge1_Node000001_Node000002" title=" " alt="" coords="137,133,137,166,132,166,132,133"/>
<area shape="rect" id="Node000003" title=" " alt="" coords="91,5,178,31"/>
<area shape="poly" id="edge2_Node000002_Node000003" title=" " alt="" coords="137,44,137,79,132,79,132,44"/>
</map>
